conceptpowerTest.py

```diff
@@ -0,0 +1,98 @@
+__author__ = 'diging_yogi'
+
+import unittest
+
+from requests.auth import HTTPBasicAuth
+from httmock import with_httmock
+import mock
+import mocks.mock_conceptpower
+import conceptpower
+
+
+class TestConceptpower(unittest.TestCase):
+
+
+
+    @mock.patch('requests.get', side_effect=mocks.mock_conceptpower.mocked_requests_search)
+    def test_search_no_concepts(self,mock_search):
+        """
+        testing the "search" method when there is no concept for
+        the lemma we are searching
+        """
+
+        conceptPower = conceptpower.Conceptpower()
+        val = conceptPower.search('Bradshaw', 0)
```

Comment:
This looks like a great start, nice and simple. The only thing I don't follow is the second positional parameter -- in the actual method the call signature is

```python
def search (self, query, pos='Noun'):
```

What is the second parameter in the mocked method? Our tests should match the signature of the actual method that we are mocking, no?